app/models/custom_field/hierarchy/item.rb

```diff
@@ -35,4 +35,6 @@ class CustomField::Hierarchy::Item < ApplicationRecord
   has_closure_tree order: "sort_order", numeric_order: true, dont_order_roots: true, dependent: :destroy
 
   scope :including_children, -> { includes(children: :children) }
+
+  def to_s = label
```

Comment:
🟡 Not sure, where this `to_s` is actually called , but from my gut feeling the string representation should be `"#{label} (#{short})" if a short is available. this is a reoccurring pattern, and we could even simplify then locations like `custom_fields_helper:184` and `hierarchy_strategy:52`.

Reply:
It is used indirectly the the grouping representers. It is used for the rows that a created for the heading groups. I don't wanted to introduce Item specific logic in the representers. If I change this to `label` and `short` it will be printed in the rows.

Reply:
But wouldn't that be a good thing?